server/execution/entities/execution.py

```diff
@@ -115,16 +115,21 @@ def add_new_player(self, role: int, vehicle: str):
         # take empty seat of vehicle
         player_to_vehicle = models.PlayersToVehicleInExecution.query.filter_by(
             execution_id=self.id,
-            vehicle_name=vehicle
+            vehicle_name=vehicle,
+            player_tan=f"empty-{vehicle}"
```

Comment:
Is the TAN part of the primary key or a unique constraint that could break with this?